harness_results.go

```diff
@@ -52,6 +52,8 @@ func (res *BenchmarkResult) Export() Test {
 		Artifacts: []TestArtifact{
 			{
 				LocalFile:        res.ArtifactPath,
+				Path:             res.ArtifactPath,
```

Comment:
I dug in a bit more looking at how files get uploaded, and I have a couple of thoughts/concerns here. 

First, we (might?) want to call filepath.Base() on this string, if users specify a filename with a lot of depth or an absolute path? It's probably safe, but I'm not sure. The inverse is probably also a concern... what if people end up having filenames with not enough information and later test runs override earlier ones?

Second, it seems like the Upload method on TestArtifact might be a good place to populate this value if it's not set. I realize this is counter to my earlier comment, but I don't think there's any good reason for that method to be so aggressive.